package.json

```diff
@@ -59,6 +60,7 @@
         "redux-devtools-extension": "^2.13.7",
         "redux-thunk": "^2.3.0",
         "ttag": "^1.5.1",
+        "vcf": "^2.0.4",
```

Comment:
`ical.js` is not enough?

Reply:
oh, that must have slipped from when I was playing with the libraries. It's not used now